README.md

```diff
@@ -38,29 +38,39 @@ or
 
 ### Configuration
 
-## Create a config.json
+### Create a config.json
 
 ```
 {
   "group_id": "1",
   "bootstrap_servers": "foo.com,bar.com",
   "topic": "messages",
-  "reject_topic": "reject_topic",
-  "schema": "<json schema>"
+  "message_timestamp": "$.jsonpath.to.timestamp",
+  "primary_keys": {
```

Comment:
primary_key**s**? How do we specify multiple primary keys?

Reply:
does this mean composite primary key ?

Reply:
yes, adding 0, 1, or more columns are all supported. Singer spec supports composite primary keys and most of the tap support it anyway.

To specify multiple columns as composite PK:
  primary_keys:
    id: "$.properties.basicTransferMetadataDto.transferId"
    version: "$.properties.basicTransferMetadataDto.transferVersion"

Reply:
In other words, it is Primary JSON Keys ?

Reply:
this means that two columns `id` and `version` will be added to the target table, the values will be extracted from the JSON message by the jsonpaths (`$.properties...`) and the two columns will be used as primary keys in the target table in target-snowflake|redshift|postgres